packages/wallets/particle/src/adapter.ts

```diff
@@ -37,12 +45,37 @@ export class ParticleAdapter extends BaseMessageSignerWalletAdapter {
     private _readyState: WalletReadyState =
         typeof window === 'undefined' ? WalletReadyState.Unsupported : WalletReadyState.Loadable;
 
-    constructor(config: ParticleAdapterConfig = {}) {
+    private _particleNetwork: ParticleNetwork | null = null;
+
+    constructor(config: ParticleAdapterConfig) {
```

Comment:
This is also a breaking change -- any app that uses `ParticleWalletAdapter` will get a type error because `config` is now a required argument.

Is there any way you can refactor this to make this not required?

Reply:
Sure, should be fixed now